dc/core/Tile.h

```diff
@@ -53,14 +53,16 @@ class Tile : public QObject
 
     Q_PROPERTY( int index READ getIndex CONSTANT )
     Q_PROPERTY( QRect coord READ getCoord NOTIFY coordChanged )
+    Q_PROPERTY( bool visible READ isVisible NOTIFY visibilityChanged )
 
 public:
     // false-positive on qt signals for Q_PROPERTY notifiers
     // cppcheck-suppress uninitMemberVar
-    Tile( const int index, const QRect& rect, QObject* parent_ = nullptr )
+    Tile( const int index, const QRect& rect, QObject* parent_, const bool isVisible_ )
```

Comment:
minor detail: I would simply name the variables "visible" instead of "isVisible" in this file